Applications/SlicerApp/Main.cxx

```diff
@@ -31,10 +31,11 @@ namespace
 {
 
 //----------------------------------------------------------------------------
-int SlicerAppMain(int argc, char* argv[])
+int
+SlicerAppMain(int argc, char * argv[])
 {
   typedef qSlicerAppMainWindow SlicerMainWindowType;
-  typedef qSlicerStyle SlicerAppStyle;
+  typedef qSlicerStyle         SlicerAppStyle;
```

Comment:
Could we turn this off? It wastes space. It makes the code harder to read. It would be very tedious to do maintain such alignment manually (without clang autoformat).

Reply:
@lassoan FYI: With this commit, the pre-commit hooks automatically fix the code, eliminating the need for developers to know and manage the rules.  It works similarly to the black pre-commit hook where the fixes are automatically applied at time of commit.

I don't care about the final format, as long as it is automatically applied.  I used ITK defaults when the were not specified in #5211.

I'll make this recommended style change in the next PR later this morning. (PS:  I prefer your suggested style as well).

Reply:
done.